### DISCUSSION TITLE
Form submission

### DISCUSSION THREAD

**Discussion by ondutyapp** [Budibase/budibase#15258] [Help] [answered]
Is it possible to (once someone has filled a form and sent it) for it to be compiled to a PDF and automatically emailed to a user?

**Comment by mikesealey** [comment, marked as answer]
Hey @ondutyapp - there are a couple of custom plugins for PDF that you can explore here: https://github.com/Budibase/plugins

There is not currently a PDF-connected aspect of Budibase as it's core. You can simply use the answers on the form as part of the email, just not as an attached PDF.  

If that's something you'd like to do, create an automation on the table that your form writes to, that gets triggered "On Row Created" - you will then have access to all of the bindings

![image](https://github.com/user-attachments/assets/2911a914-8e71-4da6-82d9-90ef84b6cabc)

It could look a little bit like this:
![image](https://github.com/user-attachments/assets/690bfc9a-6a61-4f9e-ada5-930642d8f62c)

But more consideration should be put into email styling - I've put this example together very quickly and nobody would ever confuse me with a designer.

If this doesn't cover what you're looking for, please feel free to submit a feature request [here](https://github.com/Budibase/budibase/issues/new/choose), and be sure to add your response to the vote on what you would like to see on [our 2025 Roadmap here](https://github.com/Budibase/budibase/discussions/15211)
